tests/0_check_clusterwide.robot

```diff
@@ -49,6 +50,11 @@ Check Nexus Components available
 #    List Should Contain Value               ${componentNames}           docker-hosted
 #    List Should Contain Value               ${componentNames}           raw
 
+Check Liara availability
+    [Documentation]  Check Liara UI availability
+    [Tags]  ui
+    Open Liara                              /
```

Comment:
We need at least following tests: 
1. List of enclaves present
2. Kubernetes dashboard opens